templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p>Logged out</p>
+  <a href="{% url 'login' %}">Click here to log in again!</a>

```

Comment:
The link to log in again is correctly implemented using Django's URL template tag, which ensures that the URL is dynamically generated and correct.